ui.apps/src/main/content/jcr_root/apps/saas-aem-module/components/search/search.html

```diff
@@ -1,8 +1,13 @@
 <sly data-sly-use.model="com.valtech.aem.saas.api.fulltextsearch.SearchModel"
-     data-sly-test.isEdit="${wcmmode.edit}"></sly>
+     data-sly-test.isEdit="${wcmmode.edit}"
+     data-sly-test.isConnectingToSearchApi="${model.isConnectingToSearchApi}"></sly>
+<coral-alert data-sly-test="${isEdit && !isConnectingToSearchApi}"
+             variant="warning">
+    <coral-alert-header>Can not communicate to Search API.</coral-alert-header>
```

Comment:
Interesting would be to see the reason why it failed incl. details e.g. the stracktrace. Usually connection timeout is the issue but it mustn't, could be as well missing CA configs. Can you add this & change log message to investigate logfiles? Also the request URI would be important to see.